src/app/tests/suites/certification/Test_TC_OO_1_1.yaml

```diff
@@ -35,7 +35,7 @@ tests:
       command: "readAttribute"
       attribute: "ClusterRevision"
       response:
-          value: 4
+          value: 5
```

Comment:
If we are not testing the actual contents of OnOff cluster DeadFront feature, why do we have to check "equal 5"? This will break all prior implementations that use the test scripts (you can't ever certify 1.0 or 1.1 against the test script).

There should be a test that value >= 4. Alternatively, the test to check value == 5 should only be when DeadFront feature is provided/present in PICS list. Version 5 of the cluster is not landed yet.

I think this is the first "real" cluster revision bump we see that may have wide impact.

Suggest figuring-out a way to avoid bumping everyone forcibly to version 5

Reply:
>  Version 5 of the cluster is not landed yet.

This one is particularly simple, because there are no behavior changes in version 5 other than the new feature flag.  So this PR in fact implements version 5.

Reply:
The spec does require a device to use the highest revision number in a particular release of the matter spec, so even if the test would allow to check for rev. 4 it would not really make sense if this is the highest rev number? If the device is certified for older versions of Matter, they would use the yaml files present in that version of the TH to test anyways and not the latest on master?

PCC also has bumped its rev number from 1.0 to 1.1, due to the addition of a feature map.

Reply:
>  If the device is certified for older versions of Matter, they would use the yaml files present in that version of the TH to test anyways and not the latest on master?

This is the issue: it's very unfortunate that our testing can only test "latest" and can no longer be used (without going back in history, in *branches that don't and can't exist*) against older devices at all.

Reply:
Hmm, so when you say testing, do you then think about the testing done by the CI or are you thinking about certification testing? The test harness does have its own versioning that captures a "snapshot" of the tests at that point in time, so my assumption is that we would then use these versions of TH to re-do tests on older devices?

To my understanding there is a record of which commit of the SDK goes into each TH version, so with the coming python YAML parser, it might be easier to capture the YAML test files for 1.0, 1.1 and so on as a bundle which can then be executed using the python parser, if we want to execute "old" test outside TH?

Reply:
@ReneJosefsen  QA runs backward compatibility testing every week against 1.0 and 1.0 branches. if this script is chking for only rev5 it fails the tests on 1.1 and 1.0 SDK.

Reply:
@cjandhyala, I was not aware of that. That certainly becomes an issue. Writing the test in a way that handles multiple revision does not seem like something we want to do either, though.